e2e/smoke/src/main/proto/greeting.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "smoke.grpc.generated";
+
+package smoke.grpc;
```

Buf (1.72.0)

[error] Files with package "smoke.grpc" must be within a directory "smoke/grpc" relative to root but were in directory "e2e/smoke/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)